Makefile

```diff
@@ -57,12 +61,13 @@ coverage: ${ENV_DIR}/bin/coverage cluster_mgr_setup.py queue_mgr_setup.py config
 	PYTHONPATH=src:common ${ENV_DIR}/bin/coverage run cluster_mgr_setup.py test
 	PYTHONPATH=src:common ${ENV_DIR}/bin/coverage run -a queue_mgr_setup.py test
 	PYTHONPATH=src:common ${ENV_DIR}/bin/coverage run -a config_mgr_setup.py test
+	PYTHONPATH=src:common ${ENV_DIR}/bin/coverage run -a plugins_setup.py test
```

Comment:
Yes. Should I look at setting up the right exclusions so we get 100% coverage of all we expect to be covered?